users/vieting/experiments/switchboard/transducer/feat/helpers/lr/oclr_scf.py

```diff
@@ -0,0 +1,30 @@
+def dynamic_learning_rate(*, network, global_train_step, learning_rate, **kwargs):
+    """
+    One cycle learning rate: triangular linear w.r.t. iterations (steps)
+    """
+    # -- need to be adjusted w.r.t. training -- #
+    initial_lr = 8e-5
+    peak_lr = 8e-4
+    final_lr = 1e-6
+    cycle_epoch = 135
+    total_epoch = 300
+    n_step = 2650  # steps/epoch depending on batch_size
```

Comment:
I agree that it would be better to have this configurable from the config rather than creating a new file in the recipes every time. One way would be to define some params in the config and load them here. E.g.:
```
from returnn.config import get_global_config

config = get_global_config()
initial_lr = config.typed_dict.get("learning_rate_initial_lr", 8e-5)
...
```
We could set no default value for `n_step` so one always has to set this to avoid accidental hidden changes there.

Reply:
But this would break the previous setups that used oclr without setting the n_step

Reply:
So I create the new oclr file anyway but make the new one more general? And rename it from `oclr_scf.py ` to something like `oclr_configurable.py`?

Reply:
Yes, that is my suggestion.